d2l-table-style.html

```diff
@@ -16,12 +16,12 @@
 				};
 				--d2l-table-header: {
 					color:var(--d2l-color-ferrite);
-					font-family:inherit;
-					font-size:.7rem;
-					font-weight:400;
-					line-height:1rem;
-					letter-spacing:.02rem;
-					background-color:var(--d2l-table-header-background-color);
+				      font-family:inherit;
```

Comment:
Not really :). I just opened this PR so I can test how much of a difference this makes.